docs/setup/deployment.md

```diff
@@ -6,6 +6,15 @@ deployment, **log in and change the default password immediately**.
 Many of Baby Buddy's configuration settings can be controlled using environment
 variables - see [Configuration](../configuration/intro.md) for detailed information.
 
+## Requirements
```

Comment:
Revert this change in favor of direct link to Django docs.

Reply:
Ok, this change is removed and I added database requirement for **Manual** and **Docker** installs.